xonsh/pygments_cache.py

```diff
@@ -312,9 +312,11 @@ def load_or_build():
     else:
         import sys
 
-        print("pygments cache not found, building...", file=sys.stderr)
+        if DEBUG:
+            print("pygments cache not found, building...", file=sys.stderr)
```

Comment:
Can you please put this PR into https://github.com/xonsh/pygments-cache

Reply:
https://github.com/xonsh/pygments-cache/pull/8